scripts/upload-asset.mjs

```diff
@@ -3,7 +3,7 @@ import fs from "node:fs";
 import { writeFile } from "node:fs/promises";
 import { join, parse } from "node:path";
 import { fileURLToPath } from "node:url";
-import manifest from "../src/cloud-assets/manifest.json" assert { type: "json" };
+import manifest from "../src/cloud-assets/manifest.json" with { type: "json" };
```

Comment:
https://devblogs.microsoft.com/typescript/announcing-typescript-5-3/#import-attributes

Reply:
This broke from the TS upgrade in https://github.com/membrane-io/docs/pull/23/files